drools-ansible-rulebook-integration-api/src/test/java/org/drools/ansible/rulebook/integration/api/MemoryLeakTest.java

```diff
@@ -56,4 +61,70 @@ public void testMemoryLeakWithUnmatchEvents() {
         }
         rulesExecutor.dispose();
     }
+
+    public static final String JSON_PLAIN =
+            """
+            {
+                "rules": [
+                        {
+                            "Rule": {
+                                "condition": {
+                                    "AllCondition": [
+                                        {
+                                            "EqualsExpression": {
+                                                "lhs": {
+                                                    "Event": "i"
+                                                },
+                                                "rhs": {
+                                                    "Integer": 1
+                                                }
+                                            }
+                                        }
+                                    ]
+                                },
+                                "enabled": true,
+                                "name": null
+                            }
+                        }
+                    ]
+            }
+            """;
+
+    public static final String EVENT_24KB = "{\"i\":1,\"data\":\"" + "A".repeat(24 * 1024) + "\"}";
+
+    @Ignore("This test is to check for memory leaks. It should be run manually and not as part of the build.")
```

Comment:
I set `@Ignored` for now, so CI is green. But I think I'll enable this test for CI (e.g. turtleTest) so that we can detect a memory leak earlier.

Reply:
Removed `@Ignored`